app/models/label/LabelTable.scala

```diff
@@ -390,17 +392,44 @@ object LabelTable {
         |  				LEFT JOIN sidewalk.label_severity as sev ON lb.label_id = sev.label_id
         |				  LEFT JOIN sidewalk.label_description as lab_desc ON lb.label_id = lab_desc.label_id
         |				  LEFT JOIN sidewalk.label_temporariness as lab_temp ON lb.label_id = lab_temp.label_id
-        |		  ) AS lb_big
-        |WHERE lb1.deleted = FALSE
+        |     ) AS lb_big,
+        |     (
+        |        SELECT label.label_id, text, COUNT(label_validation_id) AS count
+        |        FROM label
+        |        FULL JOIN validation_options ON TRUE
+        |        LEFT JOIN label_validation ON label.label_id = label_validation.label_id
+        |            AND label_validation.validation_result = validation_options.validation_option_id
+        |        WHERE label.deleted = FALSE
+        |        GROUP BY label.label_id, validation_option_id
+        | ) AS val
+        | WHERE lb1.deleted = FALSE
         |    AND lb1.gsv_panorama_id = gsv_data.gsv_panorama_id
         |    AND lb1.audit_task_id = at.audit_task_id
         |    AND lb1.label_id = lb_big.label_id
         |    AND at.user_id = u.user_id
         |    AND lb1.label_id = lp.label_id
-        |ORDER BY lb1.label_id DESC
-        |LIMIT ?""".stripMargin
+        |    AND lb1.label_id = val.label_id
+        | ORDER BY lb1.label_id DESC
+        | LIMIT ?""".stripMargin
     )
-    selectQuery(takeN).list.map(label => labelAndTagsToLabelMetadata(label, getTagsFromLabelId(label._1)))
+    val lst = selectQuery(takeN * 3).list
+
+    /*
+      In the query above, each label is returned with three rows, each of which are the same but 
```

Comment:
This comment is good, can you just make it so that the max line length is <120 characters instead of <100? Might save a line of vertical space. It looks sort of out of place to have just these lines being under 100 characters.